packages/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -34,20 +35,169 @@ describe('Dropdown', () => {
     })
   })
 
+  describe('open', () => {
+    const onOpenChange = jest.fn()
+
+    afterEach(() => {
+      onOpenChange.mockReset()
+    })
+
+    it('is false when closed by trigger button click', () => {
```

Comment:
```suggestion
    it('is "false" when closed by trigger button click', () => {
```

_NIT_

Reply:
will change here and for next tests as well thanks!